acceptance/bundle/deployment/bind/schema/databricks.yml

```diff
@@ -0,0 +1,10 @@
+bundle:
+  name: bind-schema-test-$BUNDLE_NAME_SUFFIX
```

Comment:
Can we add support for BUNDLE_NAME_SUFFIX into acceptance test runner and have a dedicated replacement for it?

Reply:
addressed in #2628